app/src/main/java/com/keylesspalace/tusky/service/SendStatusService.kt

```diff
@@ -138,6 +138,7 @@ class SendStatusService : Service(), Injectable {
                     statusToSend.mediaProcessed.forEachIndexed { index, processed ->
                         if (!processed) {
                             when (mastodonApi.getMedia(statusToSend.mediaIds[index]).code()) {
+                                404, // This can happen when editing posts with media that were uploaded a long time ago
```

Comment:
@connyduck Does this make any sense to you?

Reply:
No, because that codepath should only be hit when reuploading 🤔